alerter.c

```diff
@@ -11,22 +11,28 @@ int networkAlertStub(float celcius) {
     return 200;
 }
 
-void alertInCelcius(float farenheit) {
+int networkAlert(float celcius) {

```

Comment:
Is this method used , if not then it is unreachable code! needs to be cleaned up...